### ISSUE TITLE
Unclarity on how to change font of the author, "Betreuer" and "Betreuender Hochschullehrer" into serif

### ISSUE THREAD

**Issue by PhilippMatthes** [tud-cd/tudscr#73]
Hi 👋🏼 I searched the documentation and the forums but didn't find an answer to my question.

How can I change the font of the author, "Betreuer" and "Betreuender Hochschullehrer" into the serif font I am using (which is `bitstream-charter`)? For my configuration, see https://github.com/PhilippMatthes/diplom/blob/master/thesis.tex#L13

Finally, here is a screenshot of the problem

![Bildschirmfoto 2021-09-29 um 13 13 05](https://user-images.githubusercontent.com/27271818/135257654-5a6321ed-ff56-4fe4-9959-c3764154b6c5.png)

Maybe there could be a configuration command that allows to change the base font? I am by any means no LateX expert. And, maybe, the process of changing the default font could be documented.

**Comment by mrpiggi** [comment]
Just use
```latex
\addtokomafont{titlepage}{\rmfamily}
\addtokomafont{author}{\rmfamily}
```
after `\begin{document}`. 

Since you have deactivated OpenSans with `cdfont=false` all the other settings with `\addtokomafont{...}{...}` are actually not necessary.

**Comment by PhilippMatthes** [comment]
Thanks! So my mistake was to place those calls before the begin of document?

**Comment by mrpiggi** [comment]
Exactly. This is one of many items on my ToDo list. As `tudscr` does a lot for font selection and tries to derive settings from other font packages, all settings can only be applied after the preamble. That's why your configurations are actually no-op. For the next version, I'm going to delay settings with `\addtokomafont` after the font settings done by `tudscr` but currently I do not have any time for further development.
